pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -860,3 +898,60 @@ func (k *Bootstrapper) elevateKubeSystemPrivileges(cfg config.ClusterConfig) err
 	}
 	return nil
 }
+
+
+// adviseNodePressure will advise the user what to do with difference pressure errors based on their environment
+func adviseNodePressure(err error, name string, drv string) {
+	if diskErr, ok := err.(*kverify.ErrDiskPressure); ok {
+		out.ErrLn("")
+		glog.Warning(diskErr)
+		out.WarningT("The node {{.name}} has ran out of disk space.", out.V{"name": name})
+		// generic advice for all drivers
+		out.T(out.Tip, "Please free up disk or prune images.")
+		if driver.IsVM(drv) {
+			out.T(out.Stopped, "Consider creating a cluster with bigger disk size: `minikube start --disk SIZE_MB` ")
```

Comment:
This sounds more like out.Tip or out.Workaround than out.Stopped (which is used for stopped clusters)

Also, remove `Consider`: be concise and direct.

Reply:
changed wording, kept icon. till I am convinced.